.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu]
-        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2", "3.3"]
```

Comment:
Might be out of the scope adding support to Ruby 3.3 but that is the Ruby I have in my machine and the code ran, meaning it seems safe.